app/templates/administrative-units/administrative-unit/sites/site/index.hbs

```diff
@@ -0,0 +1,100 @@
+<div class="au-o-box">
+  <AuToolbar>
+    <AuToolbarGroup>
+      <AuHeading>
+        Vestiging: {{@model.sites.address.fullAddress}}
+      </AuHeading>
+    </AuToolbarGroup>
+    <AuToolbarGroup>`
```

Comment:
There is ` character there :smile: I thought it was a little insect on my screen at first :grimacing:

Reply:
ahahahahah